tests/integration/bigquery_project/definitions/example_view.sqlx

```diff
@@ -6,6 +6,12 @@ config {
             description: "val doc",
             bigqueryPolicyTags: "projects/dataform-integration-tests/locations/us/taxonomies/800183280162998443/policyTags/494923997126550963"
         }
+    },
+    bigquery: {
```

Comment:
there should be an accompanying change to the integration test assertions (there is one test case that specifically checks that table metadata stuff is set, just add to that).